### PR TITLE
LPS-179096 [PRM] Continue button is disabled

### PR DESCRIPTION
**Pull request by kiwm**
Related Issue: https://issues.liferay.com/browse/LPS-179096

### CODE REVIEW

**Comment by kiwm** [comment]
ci:test:sf

**Comment by kiwm** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/content-welcome-banner/index.html [hunk 1]
```diff
@@ -31,16 +31,16 @@ <h1 data-lfr-editable-id="first-welcome-banner-title" data-lfr-editable-type="ri
...
+	<h1 data-lfr-editable-id="second-welcome-banner-title" data-lfr-editable-type="rich-text">
 			This is editable text!
 		</h1>
 	</div>
 
 	<div class="col d-sm-flex justify-content-lg-end my-4">
-		<button class="border-brand-primary btn btn-secondary d-flex text-brand-primary video-tour-button  mr-3 mb-2">
-		<lfr-drop-zone></lfr-drop-zone>
+	<button class="border-brand-primary btn btn-secondary d-flex mb-2 mr-3 text-brand-primary video-tour-button">
+			<lfr-drop-zone></lfr-drop-zone>
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/content-welcome-banner/index.html:43]: Indentation issue!

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by claricebrasil** [comment]
ci:forward

**Pull request closed without merging**